TypeScript/devServerOnly.ts

```diff
@@ -0,0 +1,41 @@
+// @ts-nocheck
+/**
+ * Checks if the command is allowed in a specified server.
+ *
+ * @author @D3ord3NidAm [<@1017182455926624316>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { commandModule, CommandType } from "@sern/handler";
+ * import { devServerOnly } from "../plugins/devServerOnly";
+ * export default commandModule({
+ *   type: CommandType.Both,
+ *   plugins: [devServerOnly(["guildId", "guildId"], "fail message")], // fail message is the message you will see when the command is ran in the wrong server.
+ *   description: "command description",
+ *
+ *   execute: async (ctx, args) => {
+ *     // your code here
+ *   },
+ * });
+ * ```
+ */
+
+
+import { CommandType, EventPlugin, PluginType } from "@sern/handler";
+
+export function devServerOnly(
+  guildId: string[],
+  perFail: string
```

Comment:
make perFail optional, 
```ts
perFail?: string
```
As a result, would need to check if perFail is undefined. ignore silently if undefined

Reply:
Do you want a default message to send?

Reply:
up to you, however, I would rather make it optional.

Reply:
If we ignore an interaction it's just gonna output No response from the client.

Reply:
no problem, some may want that.

Reply:
So, how about I make it optional and if its a message command, just don't respond but if it is ran as an interaction, just respond with a default message?

Reply:
That's not a bad idea, go for it

Reply:
Fixed: failMessage is optional, failMessage has a default in case no message was entered.